src/App.js

```diff
@@ -1,25 +1,30 @@
-import logo from './logo.svg';
+import { BrowserRouter, useLocation } from 'react-router-dom';
 import './App.css';
+import NavBar from './nav/NavBar';
+import AppRouter from './routes/Router';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="app-container">
+      <BrowserRouter>
+        <NavBarContainer />
+        <section className="main-content">
+          <AppRouter />
+        </section>
+      </BrowserRouter>
     </div>
   );
 }
 
 export default App;
+
+function NavBarContainer() {
```

Comment:
- [ ]  Please, kindly functions in this file to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.


That way, your code will meet the latest standards. 💪